bin/build.sh

```diff
@@ -120,10 +120,10 @@ elif [ ! -d "${ANDROID_TOOLS_DIR}" ]; then
   exit 1
 fi
 
-CMD="ant clean package -debug \
-  -Dtools.dir=${ANDROID_TOOLS_DIR} \
-  -Dandroid.api.level=${ANDROID_API_LEVEL} \
-  -Dversion=${CALABASH_ANDROID_SERVER_VERSION}"
+CMD="gradle clean preparePackage \
```

Comment:
We should be using gradle-wrapper (gradlew) and not rely on a system provided gradle